src/app/_components/Icons/Facebook.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import p from 'prop-types';
+
+const FacebookComponent = ({ className }) => (
```

Comment:
@Yivgha for such cases, you can often see the usage of `svgr `in various projects. I don't have a strong opinion there; `svgr `can be tricky to set up sometimes.

You can discuss the need for `svgr ` in such cases with your colleagues.

Reply:
implemented svgr